general/development/process.md

```diff
@@ -119,13 +119,13 @@ The process of [new feature development](#new-feature-development) is described
 
 ### Testing
 
-During development, as new code is integrated, automated testing conducted at the [code](/general/development/tools/phpunit) and [interface](/general/development/tools/behat) levels, to make sure there are no regressions caused by new features.
+During development, as new code is integrated, automated testing conducted at the [code](/general/development/tools/phpunit) and [interface](./tools/behat/index.md) levels, to make sure there are no regressions caused by new features.
```

Comment:
This file still contains 4 references to /general/development (as GitHub doesn't support commenting on unchanged lines, I'm adding this general comment here.